pkg/cmd/server/start/admission_sync_test.go

```diff
@@ -16,6 +16,7 @@ import (
 var admissionPluginsNotUsedByKube = sets.NewString(
 	"AlwaysAdmit",            // from kube, no need for this by default
 	"AlwaysDeny",             // from kube, no need for this by default
+	"AlwaysPullImages",       // from kube, not enabled by default
```

Comment:
my understanding from the card was that we did not want it enabled by default.  

@ncdc @abhgupta @smarterclayton - opinion on this?